app/services/hts_service/ait_intergration/contact_csv_row_builder.rb

```diff
@@ -95,117 +93,113 @@ def age_group contact, index
         end
     end
 
-    def dob contact, index
+    def dob contact
       (Date.today - contact['Age of contact'].to_i.years).change(day: 17)
     end
 
-    def sex contact, index
-      contact['Gender of contact'].strip
-    end
-
-    def generation contact, index
+    def generation contact
       2
     end
 
-    def close_case_date contact, index
+    def close_case_date contact
       nil
     end
 
-    def registered_by contact, index
+    def registered_by contact
       csv_row_builder.registered_by contact
     end
 
-    def health_facility_id contact, index
+    def health_facility_id contact
       csv_row_builder.health_facility_id contact
     end
 
-    def health_facility_name contact, index
+    def health_facility_name contact
       csv_row_builder.health_facility_name contact
     end
 
-    def district_id contact, index
+    def district_id contact
```

Comment:
This should return the district location_id the hash currently returning an integer